Documentation/configuration.md

```diff
@@ -68,9 +68,11 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: "default=http://localhost:2380,default=http://localhost:7001"
 
 ##### -initial-cluster-state
-+ Initial cluster state ("new" or "existing").
++ Initial cluster state ("new" or "existing"). If this option is set to `new`, etcd will try to [statically bootstrap][static bootstrap]. It is safe to use `new` when starting multiple machines at the same time. If this option is set to `existing`, etcd will attempt to join the existing cluster. If the wrong value is set, etcd will attempt to start but fail safely.
```

Comment:
Set to `new` for all members _present_ during initial static or DNS bootstrapping.